Src/AutoFixture/CustomizeAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Reflection;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Base class for customizing parameters in methods.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true)]
+    public abstract class CustomizeAttribute : Attribute
```

Comment:
No, I cannot find such a scenario either. I agree that it might look a bit strange.

I noticed two attribute design patterns in AutoFixture:

1. "Lightweight" marker attributes with no logic that rely on *Relays* to create specimens. Except the [SubstituteAtribute](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoNSubstitute/SubstituteAttribute.cs#L18), all the Data Annotation customizations are implemented this way. See [RangeAttributeRelay](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/DataAnnotations/RangeAttributeRelay.cs), [RegularExpressionAttributeRelay](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/DataAnnotations/RegularExpressionAttributeRelay.cs).

2. "Heavy" attributes that perform all the fixture customization logic by themselves. That is the way all the Glue Library attributes like [Greedy](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture.xUnit.net2/GreedyAttribute.cs#L31) or [Frozen](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture.xUnit.net2/FrozenAttribute.cs) are implemented.

My intention was to make the attributes look 'similar' so that I could [manage](https://github.com/AutoFixture/AutoFixture/pull/590) them easily. I achieved this introducing the common `CustomizeAttribute` abstraction.

But it appeared that I probably need to look into the opposite way - getting rid of all the `CustomizeAttribute` classes in the Glue Libraries and replacing them with Relays. That would allow not to introduce new doubtful abstractions and prevent copy pasting attribute logic in GLs.